scripts/exgfs_atmos_fbwind.sh

```diff
@@ -56,28 +53,26 @@ export FORT13="gfs.t${cyc}z.grbf024_grb1"
 
 #       GFS grib index files
 
-export FORT31="gfs.t${cyc}z.grbif006_grb1"
-export FORT32="gfs.t${cyc}z.grbif012_grb1"
-export FORT33="gfs.t${cyc}z.grbif024_grb1"
+export FORT31="gfs.t${cyc}z.grbf006_grb1.idx"
+export FORT32="gfs.t${cyc}z.grbf012_grb1.idx"
+export FORT33="gfs.t${cyc}z.grbf024_grb1.idx"
 
 #
 #   1280 byte transmission file
 #
 
 export FORT51="tran.fbwnd_pacific"
 
-startmsg
+cp "${PARMgfs}/product/fbwnd_pacific.stnlist" fbwnd_pacific.stnlist
 
-$EXECgfs/fbwndgfs < ${PARMgfs}/product/fbwnd_pacific.stnlist >> $pgmout 2> errfile
+"${EXECgfs}/fbwndgfs.x" < fbwnd_pacific.stnlist >> "${pgmout}" 2> errfile
 export err=$?; err_chk
 
+cp tran.fbwnd_pacific "${COMOUT}/tran.fbwnd_pacific.${job_name}"
 
-cp tran.fbwnd_pacific ${COMOUTwmo}/tran.fbwnd_pacific.$job_name
-
-if test "$SENDDBN" = 'YES'
-then
-#    make_ntc_bull.pl WMOBH NONE KWNO NONE tran.fbwnd_pacific ${COMOUTwmo}/tran.fbwnd_pacific.$job_name
-   ${USHgfs}/make_ntc_bull.pl WMOBH NONE KWNO NONE tran.fbwnd_pacific ${COMOUTwmo}/tran.fbwnd_pacific.$job_name
+if [[ "${SENDDBN}" == 'YES' ]]; then
+  # Should this really be in a SENDDBN block?
```

Comment:
Not sure.  Is `make_ntc_bull.pl` calling `dbnalert`?  If so, it might be ok.

Reply:
It does, though presumably we still want to run it up until the alert point even when DBN is off.